Apps/W1/Email - Guest Outlook REST API/Test/Library-Guest-M365-REST-API/Mocks/OutlookMockInitSubscribers.Codeunit.al

```diff
@@ -0,0 +1,17 @@
+codeunit 79003 "Outlook Mock Init. Subscribers"
```

Comment:
"Guest Outlook Mock Init. Subscribers"

Not sure if that's too long, if so maybe
"Guest Outlook Mock Init. Subs."